src/main/java/org/kohsuke/github/GHMyself.java

```diff
@@ -17,6 +17,18 @@
  * @author Kohsuke Kawaguchi
  */
 public class GHMyself extends GHUser {
+
+    /**
+     * Type of repositories returned during listing.
+     */
+    public enum RepositoryType {
+        All,  // All public and private repositories that current user has access or collaborates to
```

Comment:
Enum should be CAPITAL_CASED

you can rewrite toString() method if want to change string representation